modules/books.js

```diff
@@ -0,0 +1,63 @@
+export default class Books {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.obj = { title: this.title, author: this.author };
+  }
+
+     static myCollection = [];
+
+     add(myBooksB) {
+       const books = document.createElement('tr');
+       const btn = document.createElement('button');
+       const tdBtn = document.createElement('td');
+       const td = document.createElement('td');
+       tdBtn.appendChild(btn);
+       tdBtn.classList.add('remove-btn');
+
+       myBooksB.appendChild(books);
+
+       btn.textContent = 'Remove';
+       td.textContent = `"${this.title}" by ${this.author}`;
+
+       books.appendChild(td);
+       books.appendChild(tdBtn);
+       Books.myCollection.push(this.obj);
+       localStorage.setItem('myCollection', JSON.stringify(Books.myCollection));
+       btn.addEventListener('click', () => {
+         books.remove();
+         Books.myCollection.splice(Books.myCollection.indexOf(this.obj), 1);
+         localStorage.setItem('myCollection', JSON.stringify(Books.myCollection));
+       });
+     }
+
+     static loading(myBooksB) {
+       const restore = JSON.parse(localStorage.getItem('myCollection'));
+       for (let i = 0; i < restore.length; i += 1) {
```

Comment:
- [ ] As required for the project to use ES6, kindly make sure you update the above functions/methods by using `Arrow` notation. For example:
```JavaScript
// Bad
add(myBooksB) {
  // your code here
}

static loading(myBooksB) {
  // Your Code here
}

// -------------------------------------------------------------------------
// Good
add = (myBooksB) => {
  // your code
}

static loading = (myBooksB) => {
  // Your Code here
}
```